src/layout/layout.js

```diff
@@ -191,21 +191,27 @@
    * @private
    */
   MaterialLayout.prototype.screenSizeHandler_ = function() {
```

Comment:
I think this should just be a function now to make it clear it shouldn't be accessing `this`.
```suggestion
  function screenSizeHandler_() {
```